platform/platform.go

```diff
@@ -128,6 +128,74 @@ func (t *TestCluster) DropFile(localPath string) error {
 	return nil
 }
 
+// Copy a file between two machines in a cluster.
+func TransferFile(src Machine, srcPath string, dst Machine, dstPath string) error {
```

Comment:
can this be turned into e.g. `func ReadFile(m Machine, path string) (io.ReadCLoser, error)` and combined by the caller with e.g.:

```go
rc, _ := ReadFile(fromM, "/foo")
defer rc.Close()
err = InstallFile(rc, toM, "/bar")
```